tasks/main.yml

```diff
@@ -20,5 +20,20 @@
   tags:
     - haproxy
 
+- name: Make sure service dropin directory is available
+  file:
+    path: /etc/systemd/system/marathon.service.d/
+    state: directory
+  when: ansible_service_mgr == 'systemd'
+
+- name: Add SSL settings/environment variables
+  template:
+    src: ssl.conf.j2
+    dest: /etc/systemd/system/marathon.service.d/ssl.conf
+  when: ansible_service_mgr == 'systemd'
```

Comment:
@veger still I think here you should add 'if mesos_ssl_enabled' since there is no point having empty file